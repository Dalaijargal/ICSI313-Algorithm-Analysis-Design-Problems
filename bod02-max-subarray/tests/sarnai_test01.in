100
-72 -84 75 3 -33 18 33 -82 -52 -85 11 21 49 -8 90 -58 -39 13 -40 78 -45 37 30 -71 -30 38 -62 -61 35 -50 -12 -91 6 9 55 -81 24 -74 -16 -1 -56 -42 1 -95 -75 17 25 -57 45 58 -64 -13 10 66 -98 59 -34 99 89 34 -70 63 -67 -22 -19 91 -54 -99 74 -83 48 -15 -36 -65 -87 -94 100 -29 70 61 56 60 65 57 85 -80 -78 40 84 -92 51 23 -6 53 -53 -17 80 67 7 76