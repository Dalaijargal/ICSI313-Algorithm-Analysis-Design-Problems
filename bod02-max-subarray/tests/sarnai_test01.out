680
100 -29 70 61 56 60 65 57 85 -80 -78 40 84 -92 51 23 -6 53 -53 -17 80 67 7 76 